client/src/main.ts

```diff
@@ -62,51 +123,84 @@ setupDiscordSdk().then(async () => {
 		view.msgList.pushMessage(msg)
 	})
 })
```

Comment:
Why .then() when there is top-level await?

Reply:
Because it would scream that api and app could be undefined for some reason (prob because it _could_ error) and because you did the same when initing the discord SDK